src/utils.js

```diff
@@ -59,9 +59,13 @@ export const getPaginationUrl = (response, direction, path) => {
     return null;
   }
 
-  const paginationUrl = response.links[direction];
+  let paginationUrl = response.links[direction];
   if (!paginationUrl) {
     return null;
   }
+
+  if (typeof response.links[direction] === 'object' && response.links[direction].href) {
```

Comment:
You should use `hasOwnProperties` from `utils`, otherwise lgtm

Reply:
Thanks! for the review. I have fixed it.